app/src/main/java/org/simple/clinic/newentry/PatientEntryUiRenderer.kt

```diff
@@ -19,6 +19,8 @@ class PatientEntryUiRenderer(val ui: PatientEntryUi) : ViewRenderer<PatientEntry
     val patientEntry = model.patientEntry
     identifierValueChangedCallback.pass(patientEntry.identifier.toOptional()) { renderIdentifier(patientEntry.identifier) }
 
+    model.colonyOrVillagesList?.let { ui.showColonyOrVillagesList(it) }
```

Comment:
Can we instead use an if condition along with computed properties in the model. Please refer to other ViewRenderer files

```kt
if (model.hasColonyOrVillagesList) {
  ui.showColonyOrVillagesList(model.colonyOrVillagesList!!)
}
```